website/docs/r/group.html.markdown

```diff
@@ -44,9 +44,8 @@ support for projects in this group.
 * `request_access_enabled` - (Optional) Boolean, defaults to false.  Whether to
 enable users to request access to the group.
 
-* `visibility_level` - (Optional) Set to `public` to create a public group.
-  Valid values are `private`, `internal`, `public`.
-  Groups are created as private by default.
+* `visibility_level` - (Optional) The group’s visibility. Can be `private`, `internal`, or `public`.
+The default will be the default value configured in GitLab, usually `private`.
```

Comment:
Do we need this sentence ? it makes it feel that we query gitlab to get the default.

Reply:
That is basically what happens with O+C arguments though, based on my understanding. If the value is not set then Terraform reads it and saves it ruing the next read. How would you rephrase it?

Reply:
Yes that's what happens, but I would not write the sentence.

Reply:
Cool I'll remove it